spec/features/invalidate_consent_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+describe "Invalidate consent" do
+  before { Flipper.enable(:release_1b) }
+  after { Flipper.disable(:release_1b) }
+
+  scenario "Already given" do
+    given_i_am_signed_in
+    and_consent_has_been_given
+
+    when_i_go_to_the_patient
+    then_i_see_the_consent
+
+    when_i_click_on_the_consent
+    then_i_see_the_consent
+    and_i_click_invalidate_consent
+
+    when_i_fill_in_the_notes
+    and_i_click_invalidate_consent
+    then_i_see_the_consent_has_been_withdrawn
+    and_i_cant_withdraw
```

Comment:
worth asserting here that patient session status has changed and that it's not possible to vaccinate anymore